docs/DeveloperGuide.md

```diff
@@ -689,18 +695,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
    Use case ends.
```

Comment:
For use case 7, can we say "modify a guest's bill"? Or "Add to or deduct from a guest's bill"?

Reply:
Agreed!

Reply:
I think there are still similar changes to be made within the use case steps!

Reply:
Added!